resolwe_bio/processes/workflows/wgs_gvcf.py

```diff
@@ -138,19 +127,40 @@ class Trimming:
                 required=False,
                 description="Remove low quality bases from the beginning, "
                 "if below a threshold quality.",
+                disabled="!trimming_options.enable_trimming",
             )
 
             trailing = IntegerField(
                 label="Trailing quality",
                 required=False,
                 description="Remove low quality bases from the end, if "
                 "below a threshold quality.",
+                disabled="!trimming_options.enable_trimming",
             )
 
             minlen = IntegerField(
                 label="Minimum length",
                 required=False,
                 description="Drop the read if it is below a specified length.",
+                disabled="!trimming_options.enable_trimming",
+            )
+
+        class GatkOptions:
+            """Options."""
+
+            intervals = DataField(
+                "bed",
```

Comment:
Argument `data_type` could be listed explicitly as the rest of the arguments. For example

```
data_type="bed",
```